doc/ref_model/chapters/chapter04.md

```diff
@@ -278,47 +278,55 @@ The idea of the Cloud Infrastructure profiles is to have a predefined set of inf
 <p align="center"><b>Figure 4-2:</b> Workloads built against Cloud Infrastructure Profiles and compute Flavours.</p>
 
 <a name="4.2.1"></a>
-### 4.2.1 Compute Flavours
+### 4.2.1 Workload Profiles
 
-Compute Flavours represent the compute, memory, storage, and management network resource templates that are used by VMs on the compute hosts. Each VM is given a compute Flavour (resource template), which determines the VMs compute, memory and storage characteristics.
+Compute Flavours represent the compute, memory, storage, and management network resource sizing templates that are used to create VMs on the compute hosts. Each VM create request specifies a Workload Profile and a compute Flavour, which will determine the compute host (node) on which the VM will be hosted and the VMs compute, memory and storage characteristics.
+
+Parameterized Compute Flavours can be used by operators and specified in request for resources. 
 
 Compute Flavours can also specify secondary ephemeral storage, swap disk, etc. A compute Flavour geometry consists of the following elements:
 
 <a name="Table4-12"></a>
 
-| Element                               | Description                                                                                                                                                                                             |
+| Element                               | Description  |
 |---------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Compute Flavour Name                  | A descriptive name                                                                                                                                                                                      |
-| Number of vCPUs | Number of virtual compute resources (vCPUs) presented to the VM instance.                                                                                                                               |
-| Memory                                | Virtual compute instance memory in megabytes.                                                                                                                                                           |
+| Compute Flavour Name                  | A descriptive name |
+| Compute Flavour ID                  | Flavor unique identifier |
```

Comment:
Do compute flavour name and ID apply to containers?
I would prefer to stick with resources sizing

Reply:
```suggestion
```